src/sentry_plugins/pushover/client.py

```diff
@@ -6,9 +6,10 @@ class PushoverClient(ApiClient):
     allow_redirects = False
     plugin_name = "pushover"
 
-    def __init__(self, userkey=None, apikey=None):
+    def __init__(self, userkey=None, apikey=None, project_id=None):
         self.userkey = userkey
         self.apikey = apikey
+        self.integration_id = project_id
```

Comment:
Hmm I get that you're calling this the integration ID for consistency with the other stuff, but would it mess anything up if we just called this `project_id`?

Reply:
I don't think it should cause any problems and made the change